sklearn/neighbors/_classification.py

```diff
@@ -111,25 +111,12 @@ class KNeighborsClassifier(KNeighborsMixin, ClassifierMixin, NeighborsBase):
         False when `y`'s shape is (n_samples, ) or (n_samples, 1) during fit
         otherwise True.
 
-    Examples
-    --------
-    >>> X = [[0], [1], [2], [3]]
-    >>> y = [0, 0, 1, 1]
-    >>> from sklearn.neighbors import KNeighborsClassifier
-    >>> neigh = KNeighborsClassifier(n_neighbors=3)
-    >>> neigh.fit(X, y)
-    KNeighborsClassifier(...)
-    >>> print(neigh.predict([[1.1]]))
-    [0]
-    >>> print(neigh.predict_proba([[0.9]]))
-    [[0.66666667 0.33333333]]
-
     See Also
     --------
-    RadiusNeighborsClassifier
-    KNeighborsRegressor
-    RadiusNeighborsRegressor
-    NearestNeighbors
+    RadiusNeighborsClassifier: Classifier among neighbors within a given radius.
```

Comment:
It might be better to keep the same description than below the regressor below

```
based on neighbors within a fixed radius.
```

Reply:
I agree! Will change it now...